docs/update-locking.md

```diff
@@ -16,26 +16,59 @@ assumption that updates are locked (see [issue #20](https://github.com/resin-io/
 
-### Taking the lock
+### Creating the lockfile
 
-Using the above-mentioned library, the lock can be acquired like in this CoffeeScript example:
+There are many different tools and libraries that provide proper lockfile functionality. There are many different tools and libraries to provide proper lockfile functionality and a few common examples are shown below.
+
+__Note:__ Just creating the lockfile, for example by using `touch /tmp/resin/resin-updates.lock`, is not adequate to prevent updates. A file created in this way won't have the exclusive access flag set, and thus does not provide reliable locking.
+
+#### Shell
+
+One simple way to create a lockfile is using [lockfile](https://linux.die.net/man/1/lockfile) (available for example in Debian from the `procmail` package):
+
+```shell
+lockfile /tmp/resin/resin-updates.lock
+# ... (do things)
+rm -f /tmp/resin/resin-updates.lock
+```
+
+Another tool is [flock](https://linux.die.net/man/1/flock) (available for example in Debian from the `linux-utils` package):
+
+```shell
+flock /tmp/resin/resin-updates.lock -c '... (command to run while locked)'
+```
+
+For more examples and explanation of the functionality, check the links to the specific tools above.
+
+#### Javascript and Coffeescript
+
+Using the [`lockfile` library](https://www.npmjs.com/package/lockfile), the lock can be acquired like in this CoffeeScript example:
 ```coffeescript
-	lockFile = require 'lockfile'
+lockFile = require 'lockfile'
 
-	lockFile.lock '/tmp/resin/resin-updates.lock', (err) ->
-		# A non-null err probably means the supervisor is about to kill us
-		throw new Error('Could not acquire lock: ', err) if err?
+lockFile.lock '/tmp/resin/resin-updates.lock', (err) ->
+	# A non-null err probably means the supervisor is about to kill us
+	throw new Error('Could not acquire lock: ', err) if err?
 
-		# Here we have the lock, so we can do critical stuff:
-		doTheHarlemShake()
+	# Here we have the lock, so we can do critical stuff:
+	doTheHarlemShake()
 
-		# Now we release the lock, and we can be killed again
-		lockFile.unlock '/tmp/resin/resin-updates.lock', (err) ->
-			# If err is not null here, something went really wrong
-			throw err if err?
+	# Now we release the lock, and we can be killed again
+	lockFile.unlock '/tmp/resin/resin-updates.lock', (err) ->
+		# If err is not null here, something went really wrong
+		throw err if err?
 ```
 
-There are other libraries you can use in different languages, for example [this Python library](http://pythonhosted.org/lockfile/lockfile.html#examples).
+#### Python
+
+In Python you can use the [`lockfile` library](http://pythonhosted.org/lockfile/lockfile.html#examples)
+```python
+from lockfile import LockFile
+lock = LockFile("/tmp/resin/resin-updates.lock")
+with lock:
+    print lock.path, 'is locked.'
+```
+Check the link for more examples and other Python libraries that provide locking.
 
 ### Overriding the lock
 
```

Comment:
While we're at it, could you change the name of the configuration variable below to `RESIN_SUPERVISOR_OVERRIDE_LOCK` and make it clear that it's a configuration variable instead  of an env var?

The old `RESIN_OVERRIDE_LOCK` is actually deprecated (though for now it still works for backwards-compat).

Reply:
@pcarranzav the value is set to `1` or `true`? I see that the multicontainer demo uses true, and I wonder, which is better to say.